src/fullnode.rs

```diff
@@ -319,6 +320,10 @@ impl Fullnode {
             tick_height,
         );
 
+        while self.bank.tick_height() < tick_height {
+            sleep(Duration::from_millis(10));
```

Comment:
There's gotta be a smarter way to do this rather than a semi-random sleep too.

Reply:
Sure, but that only further justifies a test. A smarter way can be postponed if we add a good test now.